src/app/util/chip-message-send.cpp

```diff
@@ -106,18 +120,15 @@ EmberStatus chipSendUnicast(NodeId destination, EmberApsFrame * apsFrame, uint16
     // receive the ack message. This logic needs to be deleted after we convert all legacy ZCL messages to IM messages.
     DeviceExchangeDelegate delegate;
     exchange->SetDelegate(&delegate);
+
     Messaging::SendFlags sendFlags;
 
     sendFlags.Set(Messaging::SendMessageFlags::kFromInitiator).Set(Messaging::SendMessageFlags::kNoAutoRequestAck);
```

Comment:
Do we still need to disable CRMP after use same exchange to send response?

Reply:
This is the codepath where we're sending to a specific node by node id, so still creating a temporary exchange.  For attribute reporting, for example.  To stop disabling CRMP here we would need to have a nontrivial exchange delegate that actually handles the messages from the other side properly and closes the exchange as needed.  We can do that, but I was trying to not scope-creep this PR.